returnn/search.py

```diff
@@ -88,7 +90,9 @@ def __init__(
 
         self.out_returnn_config_file = self.output_path("returnn.config")
 
-        self.out_search_file = self.output_path("search_out")
+        self.out_search_file = self.output_path(
+            f"search_out" + (f".{output_mode}.gz" if output_gzip else "")
```

Comment:
```suggestion
            "search_out" + (f".{output_mode}.gz" if output_gzip else "")
```

no f-string required but not something critical ^^